db/db.go

```diff
@@ -1,25 +1,15 @@
-package main
+package db
 
 import (
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
-	"log"
 	"net/http"
 	"strconv"
 
 	"github.com/gorilla/mux"
 )
 
-type TournamentInfo struct {
-	id      int
-	name    string
-	deposit int
-	prize   int
-	users   []int
-	winner  string
-}
-
 type User struct {
```

Comment:
Move `User` struct with json tags to the server.go.
In the db package you can create its own `User` struct without tags.

Reply:
Hint:
You can create `user.go` in this package with `type User struct`